src/app/public/src/modules/shared/search-results-provider.spec.ts

```diff
@@ -0,0 +1,391 @@
+import { expect } from '@blackbaud/skyux-builder/runtime/testing/browser';
+import { Response, ResponseOptions } from '@angular/http';
+import { Observable } from 'rxjs';
+
+import { StacheSearchResultsProvider, StacheConfigService } from './index';
+
+class MockHttpService {
+  public post = jasmine.createSpy('post').and.callFake((url: string, body: any) => {
+    console.log(url);
+    console.log(body);
```

Comment:
Logs needed?

Reply:
Yes. I'm spying on those to check values.